pkg/webui/console/store/middleware/logics/events.js

```diff
@@ -211,8 +203,9 @@ const createEventsConnectLogics = (reducerName, entityName, onEventsStart) => {
         const status = selectEntityEventsStatus(getState(), id)
         const connected = status === CONNECTION_STATUS.CONNECTED
         const interrupted = selectEntityEventsInterrupted(getState(), id)
-        if (!connected && interrupted) {
+        if (!connected || interrupted) {
```

Comment:
The comment above does not match the condition, and this would still always `allow` due to lack of `return`.